src/domains/sign/key_vault.tf

```diff
@@ -14,7 +14,8 @@ module "key_vault_secrets" {
     "SelfCareApiKey",
     "SlackWebhookUrl",
     "LollipopPrimaryApiKey",
-    "LollipopSecondaryApiKey"
+    "LollipopSecondaryApiKey",
+    "PdvTokenizerApiKey"
   ]
```

Comment:
We can also remove `TokenizerApiSubscriptionKey`

Reply:
Yes, it can be removed